README.md

```diff
@@ -266,6 +266,10 @@ By default, the deployed Azure web app will only allow requests from the same or
 
 For the frontend code, change `BACKEND_URI` in `api.ts` to point at the deployed backend URL, so that all fetch requests will be sent to the deployed backend.
 
+For an alternate frontend that's written in Web Components and deployed to Static Web Apps, check out
```

Comment:
@anfibiacreativa Added shoutout to your repo here

Reply:
Thank you!